debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -696,6 +708,27 @@ protected String pluginName() {
         return config.getString(PLUGIN_NAME);
     }
 
+    protected MessageDecoder pluginDecoder() {
```

Comment:
This is is intentional. I wanted to provide the user to modify the decoder in case of a necessity. If it is not provided it is derived from plugin name in method createDefaultMessageDecoder. But I'll extract both of the values into a custom enum.

Reply:
Got you; I'd still prefer to add this option only once there's actual demand for it. Let's keep things simple, for our own sake but also in terms of exposing as less knobs and options as possible.